NEWS

```diff
@@ -28,8 +28,8 @@ $HEADER$
 
 ===========================================================================
 
-This file contains the main features added, changed behavior, and legacy report
-removed for each version of Open MPI since version 1.0.
+This file contains the main features added, changed behavior, and legacy features
+removed from each version of Open MPI since version 1.0.
```

Comment:
This feels wrong to me; perhaps something like:

"This file contains notable changes between releases for each release since Open MPI 1.0.  Because Open MPI maintains multiple release branches at any time, it is possible for items to appear more than once in the list."

Reply:
that sounds good.  i'll change the wording.